registry/nacos/registry.go

```diff
@@ -169,36 +169,95 @@ func (nr *nacosRegistry) Subscribe(url *common.URL, notifyListener registry.Noti
 	if role != common.CONSUMER {
 		return nil
 	}
+	serviceName := getServiceName(url)
+	if serviceName == "*" {
+		// Subscribe to all services
+		for {
+			if !nr.IsAvailable() {
+				logger.Warnf("event listener game over.")
+				return perrors.New("nacosRegistry is not available.")
+			}
 
-	for {
-		if !nr.IsAvailable() {
-			logger.Warnf("event listener game over.")
-			return perrors.New("nacosRegistry is not available.")
-		}
+			services, err := nr.getAllSubscribeServiceNames()
```

Comment:
Do we need a scheduled pulling task here? What if the service name list in Nacos server changes after we subscribed to '*', for example, if a new service name is registered, will we still get notified by subscribing to '*'?

Reply:
This can be added later.